components/BillTestimonies/BillTestimonies.js

```diff
@@ -13,11 +13,10 @@ const ViewTestimonies = (props) => {
   const testimonies = testimoniesResponse.status == "success" ? testimoniesResponse.result : []
 
   const testimoniesComponent = testimonies.map((testimony, index) => {
-      // need to hide author display name if testimony is anonymous
       return (
         <tr key={index}>
           <td>{testimony.position}</td>
-          <td>{testimony.authorDisplayName == null ? "John Doe" : testimony.authorDisplayName }</td>
+          <td>{testimony.authorDisplayName == null ? "Test" : testimony.authorDisplayName }</td>
```

Comment:
Consider using "Anonymous"

Reply:
I thought about "Anonymous" but then I remembered we decided not to allow anonymous testimony (for now) and I thought that might be confusing.  I think these are just testimonies that were created before we started saving the user information with the testimony.

Reply:
We could also resolve this by making sure all the sample testimonies have a uId.  This will be enforced by the software going forward.